testing/trino-benchto-benchmarks/pom.xml

```diff
@@ -83,7 +83,7 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
-                <version>1.2.5.RELEASE</version>
+                <version>2.7.1</version>
```

Comment:
ouch .. spring boot.. thats a big jump .. but looks like it all works..

Reply:
it works. it's just some conventions. this is the minimal plugin version that support reproducible builds